content/blog/2024-02-29-announcing-aldg.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "What to expect at AlmaLinux Day: Germany"
+type: blog
+author: 
+ name: "benny Vasquez"
+ bio: "Chair, AlmaLinux OS Foundation"
+ image: /users/benny.jpeg
+date: 2024-02-29
+images:
+  - /blog-images/2024/aldg.png
+post: 
+    title: "What to expect at AlmaLinux Day: Germany"
+    image: /blog-images/2024/aldg.png
+---
+
+In December, we hosted the first [AlmaLinux Day in Tokyo](/aldt-2023/). As a user-focused event that brought together our community from all over Japan, this wildly successful event focused on the needs and desires of the open source community in Japan. We had nearly 100 attendees and 7 speakers from different industries!
+
+We're excited to build on that momentum with AlmaLinux Day: Germany, coming to Europa Park Rust on March 18th. 
```

Comment:
Maybe hyperlink the first reference of `AlmaLinux Day: Germany` to the official landing page (https://339c2f2b.almalinux-org.pages.dev/almalinux-day-germany-2024/)?

I'm unsure on this.

Reply:
1000000%. Good catch!